packages/babel-preset-pwa-kit/package.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "babel-preset-pwa-kit",
+    "version": "2.0.0-dev.6",
+    "engines": {
+      "node": "^12.0.0 || ^14.0.0",
+      "npm": "^6.14.4 || ^7.0.0 || ^8.0.0"
+    },
+    "description": "Babel preset used by PWA Kit",
+    "repository": {
+      "type": "git",
+      "url": "git+https://github.com/SalesforceCommerceCloud/pwa-kit.git"
+    },
+    "homepage": "https://github.com/SalesforceCommerceCloud/pwa-kit/tree/develop/packages/babel-preset-pwa-kit#readme",
+    "bugs": {
+      "url": "https://github.com/SalesforceCommerceCloud/pwa-kit/issues"
+    },
+    "license": "SEE LICENSE IN LICENSE",
+    "author": "dev@mobify.com",
+    "main": "index.js",
+    "scripts": {
+      "lint": "npm run lint:js",
+      "lint:fix": "npm run lint:js -- --fix",
+      "lint:js": "cross-env ../../node_modules/.bin/eslint \"**/*.{js,jsx}\""
+    },
+    "dependencies": {
+        "@babel/plugin-proposal-object-rest-spread": "^7.4.4",
+        "@babel/plugin-transform-async-to-generator": "^7.7.0",
+        "@babel/plugin-transform-modules-commonjs": "^7.4.4",
```

Comment:
We're not using the `@babel/plugin-transform-modules-commonjs` plugin in the `babel-preset-pwa-kit` package. 
Do we need to include it as a dependency?